Dockerfile

```
@@ -0,0 +1,13 @@
+FROM python:3.7
+COPY . /workdir
+WORKDIR /workdir
+RUN pip install --upgrade pip && pip install \
+    black \
+    codecov \
+    flake8 \
+    mutmut \
+    pandas \
+    pylint \
+    pytest-cov \
+    pytest==5.0.1 
+CMD make
```

Por favor podrías agregar fin de línea a la última línea?
```suggestion
CMD make

```